core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1920,20 +1985,22 @@ oppia.constant('STATE_ERROR_MESSAGES', {
   ADD_INTERACTION: 'Please add an interaction to this card.',
   STATE_UNREACHABLE: 'This card is unreachable.',
   UNABLE_TO_END_EXPLORATION: (
-    'There\'s no way to complete the exploration starting from this card.')
+    'There\'s no way to complete the exploration starting from this card.'),
+  INCORRECT_SOLUTION: (
+    'This solution does not lead to another state.')
 });
 
 // Service for the list of exploration warnings.
 oppia.factory('explorationWarningsService', [
```

Comment:
Just to check -- is it possible for there to be an interim state of affairs where a solution exists but no explanation exists (e.g. in a draft prior to publication)? If so, are warnings needed for this case?